config/locales/de.yml

```diff
@@ -322,6 +322,54 @@ de:
       title: 'Titel'
       teacher: 'DozentIn'
       editors: 'EditorInnen'
+    annotation:
+      annotations: 'Annotationen'
+      annotation_modal_head: 'Annotation anlegen'
+      inherit_from_lecture: 'von Vorlesung erben'
+      time: 'Zeit:'
+      comment: 'Kommentar'
+      color: 'Farbe'
+      category: 'Kategorie'
+      note: 'Notiz'
+      content: 'Inhalt'
+      mistake: 'Fehler'
+      presentation: 'Darstellung'
+      where_problem: >
+        Wo liegt das Problem?
+      definition: 'Definition'
+      argument: 'Argument'
+      strategy: 'Beweisstrategie'
```

Comment:
As a user, I'm overwhelmed with all these category options to choose from, for example I ask myself questions such as:
- What does "presentation" mean?
- What does "argument" mean for "where's the problem"? "Argument" is such a broad word.
- Why do I get a "Where's the problem" to choose from, but then there's also the option "mistake", which does not give me the "Where's the problem?" options. What's the difference between a "content" mistake and a "mistake"?
- Why do I only get a help link when I click on "content", then on "definition". Why not also for the other options, like "proof strategy"? Why not for the high-level option "mistake"?

I feel like we definitely need some help texts for the users here. Or even restructure it a tiny bit, e.g. remove "mistake" as top-level option.